storage/migrations/02_runtimes.up.sql

```diff
@@ -34,18 +34,34 @@ CREATE TABLE chain.runtime_transactions
   round       UINT63 NOT NULL,
   FOREIGN KEY (runtime, round) REFERENCES chain.runtime_blocks DEFERRABLE INITIALLY DEFERRED,
   tx_index    UINT31 NOT NULL,
+  timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
+
   tx_hash     HEX64 NOT NULL,
   tx_eth_hash HEX64,
-  gas_used UINT63 NOT NULL,
+  -- NOTE: The signer(s) and their nonce(s) are stored separately in runtime_transaction_signers.
+
+  fee         UINT_NUMERIC NOT NULL,
+  gas_limit   UINT63 NOT NULL,
+  gas_used    UINT63 NOT NULL,
+
   size UINT31 NOT NULL,
-  timestamp TIMESTAMP WITH TIME ZONE NOT NULL,
+  
+  -- Transaction contents.
+  method      TEXT NOT NULL,  -- accounts.Transter, consensus.Deposit, consensus.Withdraw, evm.Create, evm.Call
+  body        JSON NOT NULL,  -- For EVM txs, the EVM method and args are encoded in here.
```

Comment:
is this EVM only? or is the comment about this having a different meaning in some cases?